cmd/validate/image_test.go

```diff
@@ -1025,6 +1096,74 @@ func Test_WarningOutput(t *testing.T) {
 	  }`, effectiveTimeTest, utils.TestPublicKeyJSON, utils.TestPublicKeyJSON), out.String())
 }
 
+func Test_FailureImageAccessibilityNonStrict(t *testing.T) {
+	validate := func(_ context.Context, component app.SnapshotComponent, _ policy.Policy, _ bool) (*output.Output, error) {
+		return &output.Output{
+			ImageSignatureCheck: output.VerificationStatus{
+				Passed: true,
+			},
+			ImageAccessibleCheck: output.VerificationStatus{
+				Passed: false,
+				Result: &evaluator.Result{Message: "Image URL is not accessible: HEAD http://${REGISTRY}/v2/acceptance/does-not-exist/manifests/latest: unexpected status code 404 Not Found (HEAD responses have no body, use GET for details)"},
```

Comment:
Would be better to match the value passed via `--image`
```suggestion
				Result: &evaluator.Result{Message: "Image URL is not accessible: HEAD registry/image:tag: unexpected status code 404 Not Found (HEAD responses have no body, use GET for details)"},
```